index.js

```diff
@@ -1,13 +1,13 @@
 'use strict'
 
-var window = require('global/window')
-
-var OfflineContext = window.OfflineAudioContext || window.webkitOfflineAudioContext
-var Context = window.AudioContext || window.webkitAudioContext
-
 var cache = {}
 
 module.exports = function getContext (options) {
+	if (!window) return null
```

Comment:
We cannot do like that in nodejs, it should be `typeof window === 'undefined'` then